src/submission/submission.service.ts

```diff
@@ -116,6 +119,33 @@ export class SubmissionService {
     );
   }
 
+  async findUsersByQuestionIds(
+    questionIds: number[],
+    page: number = 1,
+    limit: number = 10
+  ): Promise<User[]> {
+    const submissions = await this.submissionRepository.find({
```

Comment:
This should be enough to fetch users as attached relations just add pagination here.